packages/victory-vendor/scripts/build.js

```diff
@@ -80,16 +80,14 @@ const main = async () => {
   const EsmBasePath = path.resolve(__dirname, `../es`);
   const CjsBasePath = path.resolve(__dirname, `../lib`);
   const VendorBasePath = path.resolve(__dirname, `../lib-vendor`);
+  const baseDirs = [EsmBasePath, CjsBasePath, VendorBasePath];
+  const cleanGlobs = [].concat(baseDirs, path.resolve(__dirname, "../d3-*"));
```

Comment:
I noticed that when running `nps clean.all` (at the root), it doesn't clean the `lib-vendor` folder (since it's just hard-coded to clean `es, lib` folders).  Would it be much work to include `lib-vendor` when cleaning all?

Reply:
Ah, I should add `--ignore victory-vendor` to the root `package-scripts.js`. We're off the main path here and our nps/lerna setup isn't great for easy overrides of just small things, so the build script is the right place to handle this for vendor.

Reply:
And we also need to clean `victory-vendor/d3-*` files too, so I don't think we want to shove all that in root task.